### PR TITLE
Add Stylelint as checkstyle reporter

### PR DESCRIPTION
**Pull request by NotMyFault**
The change proposed adds stylint as alias for the checkstyle reporter. Follow up to https://github.com/jenkinsci/analysis-model/pull/812

We _may_ need an update of analysis-mode plus its api here, I'm not too sure about it.

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did

### CODE REVIEW

**Comment by uhafner** [comment]
You need to merge in https://github.com/jenkinsci/warnings-ng-plugin/pull/1310

File: plugin/src/main/java/io/jenkins/plugins/analysis/warnings/StyleLint.java
Comment (uhafner) [inline plugin/src/main/java/io/jenkins/plugins/analysis/warnings/StyleLint.java:45]: Is there an SVG available as well?

Can you please check the license of the icon and add it to https://github.com/jenkinsci/warnings-ng-plugin/blob/master/plugin/src/main/webapp/icons/LICENSE.txt?

**Review by uhafner — CHANGES_REQUESTED** [review]

File: plugin/src/main/java/io/jenkins/plugins/analysis/warnings/StyleLint.java
Comment (NotMyFault) [reply plugin/src/main/java/io/jenkins/plugins/analysis/warnings/StyleLint.java:45]: > Is there an SVG available as well?

Yes, should I add it instead of the .png ones? I was under the impression they are needed.

**Comment by NotMyFault** [comment]
> You need to merge in #1310

Oic, thanks a bunch of the quick release. I'll merge it once it's merged into master 👍🏻

File: plugin/src/main/java/io/jenkins/plugins/analysis/warnings/StyleLint.java
Comment (uhafner) [reply plugin/src/main/java/io/jenkins/plugins/analysis/warnings/StyleLint.java:45]: Not anymore. Some tools still have no SVG so both options will work. You need to use `SvgIconLabelProvider`.

**Comment by NotMyFault** [comment]
> > You need to merge in #1310
> 
> Oic, thanks a bunch of the quick release. I'll merge it into this PR once it's merged into master 👍🏻

There we go, checks are all green.

**Review by uhafner — APPROVED** [review]

**Comment by NotMyFault** [comment]
Hey,

would you mind cutting a release please? I'm looking forward to use the recent addition on ci.j 👀 

Thanks in advance!

**Comment by uhafner** [comment]
https://github.com/jenkinsci/warnings-ng-plugin/releases/tag/v9.14.0

**Pull request merged**
